conv/main.go

```diff
@@ -6,10 +6,12 @@ const (
 	places int32 = 6
 )
 
+// FloatToDecimal convert float to decimal
 func FloatToDecimal(input float64) decimal.Decimal {
 	return decimal.NewFromFloat(input).Truncate(places)
 }
 
+// StringToDecimal convert number in string to decimal
```

Comment:
mb string number? what you think?

Reply:
nope 
<img width="614" alt="2018-01-29 18 40 51" src="https://user-images.githubusercontent.com/19668889/35518932-f41cf8ac-0523-11e8-8703-ee9ad458f5d5.png">

Reply:
hm.. ok